site/reference/schema/kptfile/README.md

```diff
@@ -0,0 +1,140 @@
+# Example kptfile
+
+```yaml
+# API version of the Kptfile.
+apiVersion: kpt.dev/v1alpha1
+
+# Always "kptfile."
+kind: Kptfile
+
+# Metadata of the resource described by the kptfile.
+metadata:
+    # Name of this resource.
+    name: resource-name
+
+    # Namespace of this resource.
+    namespace: resource-namespace
+
+    # Labels used to optionally describe the resource.
+    labels:
+        app: hello
+
+    # Annotations used to optionally describe the resource.
+    annotations:
+        config.kubernetes.io/local-config: "true"
+
+
+# The upstream locator for a package.
+upstream:
+    # The type of origin for a package. Only supports "git."
+    type: git
+
+    # The locator for a package on Git.
+    git:
+        # The repository of the package.
+        repo: myrepo
+        # The sub directory in the git repository.
+        directory: /mydir/mysubdir
+        # A git branch, tag, or a commit SHA-1.
+        ref: de3c08
+
+    # The method used in updating from upstream.
+    # Supports "resource-merge," "fast-forward" and "force-delete-replace."
+    updateStrategy: resource-merge
+
+# The resolved locator for the last fetch of the package.
```

Comment:
Since this is not used by the end users, I will move this to the bottom of this doc.

Reply:
Done.